modules/osdk-init-resource.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-init-resource_{context}"]
+= Initializing required custom resources
+
+An Operator might require the user to instantiate a custom resource before the
+Operator can be fully functional. However, it can be challenging for a user to
+determine what is required or how the resource should be defined.
+
+By adding the `operatorframework.io/initialization-resource` annotation in the
+ClusterServiceVersion (CSV), an Operator developer can specify a required custom
+resource that should be created at the time that the Operator is installed. The
+annotation must include a template that contains a complete YAML initialization
+of the resource to be used during the installation process.
+
+For example, the following annotation requires that a `StorageCluster` resource
```

Comment:
maybe s/that a `StorageCluster` resource must be created/the creation of a `StorageCluster` resource